Week3/school_dump.sql

```diff
@@ -0,0 +1,247 @@
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+--
+-- Table structure for table `administration`
+--
+
+DROP TABLE IF EXISTS `administration`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `administration` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(50) NOT NULL,
+  `task` varchar(50) NOT NULL,
+  `address` varchar(200) DEFAULT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `administration`
+--
+
+LOCK TABLES `administration` WRITE;
+/*!40000 ALTER TABLE `administration` DISABLE KEYS */;
+INSERT INTO `administration` VALUES (1,'Maartje','Educational Director','1221BB Amsterdam');
+/*!40000 ALTER TABLE `administration` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `class`
+--
+
+DROP TABLE IF EXISTS `class`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `class` (
+  `id` int(11) NOT NULL,
+  `name` varchar(9) DEFAULT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `class`
+--
+
+LOCK TABLES `class` WRITE;
+/*!40000 ALTER TABLE `class` DISABLE KEYS */;
+INSERT INTO `class` VALUES (6,'6A');
+/*!40000 ALTER TABLE `class` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `office`
+--
+
+DROP TABLE IF EXISTS `office`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `office` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `admin_id` int(11) NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `admin_id` (`admin_id`),
+  CONSTRAINT `office_ibfk_1` FOREIGN KEY (`admin_id`) REFERENCES `administration` (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `office`
+--
+
+LOCK TABLES `office` WRITE;
+/*!40000 ALTER TABLE `office` DISABLE KEYS */;
+INSERT INTO `office` VALUES (1,1);
+/*!40000 ALTER TABLE `office` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `parent`
+--
+
+DROP TABLE IF EXISTS `parent`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `parent` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(50) DEFAULT NULL,
+  `relationship` varchar(50) NOT NULL,
+  `address` varchar(200) DEFAULT NULL,
+  `student_id` int(11) NOT NULL,
+  PRIMARY KEY (`id`),
+  CONSTRAINT `parent_ibfk_1` FOREIGN KEY (`id`) REFERENCES `student` (`id`)
```

Comment:
This assumes each parent can only be the parent of one student, but what if two or more siblings go to the same school? I think this should be a many-many relationship, but you've modeled it as one-many here. Think - one student can have many parents, but one parent can have many students.

If you think one parent can have many students, but each student can only have one parent, then you can make it a one-many relationship - but in the other direction, where each 'student' record contains a reference to a parent.